packages/plugin/package.json

```diff
@@ -4,6 +4,13 @@
   "repository": "https://github.com/dotansimha/graphql-eslint",
   "author": "Dotan Simha <dotansimha@gmail.com>",
   "license": "MIT",
+  "description": "GraphQL plugin for ESLint",
+  "keywords": [
```

Comment:
good for npm https://eslint.org/docs/developer-guide/working-with-plugins#share-plugins